src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -76,6 +76,11 @@ public TimeSpan AutomaticRefreshInterval
         /// </summary>
         public BaseConfiguration LKGConfiguration { get; set; }
 
+        /// <summary>
```

Comment:
Should we use the full name, such as: LastKnownGood instead of LKG?

Reply:
I changed it to LastKnownGood... hoping this way the meaning of this feature will be far more clear for developers if they don't read the documentation.